src/components/Tabs/Tabs.tsx

```diff
@@ -134,10 +134,9 @@ class TabsInner extends PureComponent<CombinedProps, State> {
       newDesignLanguage && styles.newDesignLanguage,
     );
 
-    const disclosureButtonClassName = classNames(
-      styles.DisclosureActivator,
-      hasCustomDisclosure && styles.Tab,
-    );
+    const disclosureButtonClassName = hasCustomDisclosure
```

Comment:
I'm confused by this change - can you help explain @ginabak ?

Reply:
YES, I was playing around at first with the tabs (trying to find where the error was) - thinking about - you are completely correct in being confused. I forgot to revert it back 🙈, thank-you.